collector/compile-benchmarks/README.md

```diff
@@ -196,25 +197,21 @@ Rust code being written today.
       `"touch_file"` entry.
     - See [`collector/src/benchmark/mod.rs`](https://github.com/rust-lang/rustc-perf/blob/12cb796f8a932a891b385ba23a36d78a2867ace1/collector/src/benchmark/mod.rs#L24-L27) for a complete reference.
   - Consider adding one or more `N-*.patch` files for the `IncrPatched`
-    scenario.
-    - If it's a primary benchmark, you should definitely do this.
-    - These usually consist of a patch that adds a single
-      `println!("testing");` statement somewhere.
-    - Creating the patch against what you've committed so far might be useful.
-      Use `git diff` from the repository root, or `git diff --relative` within
-      the benchmark directory. Copy the output into the `N-*.patch` file.
-    - Do a test run with an `IncrPatched` scenario to make sure the patch
-      applies correctly, e.g. `target/release/collector bench_local +nightly
-      --id Test --profiles=Check --scenarios=IncrPatched
-      --include=$NEW_BENCHMARK`
+    scenario. - If it's a primary benchmark, you should definitely do this. - These usually consist of a patch that adds a single
+    `println!("testing");` statement somewhere. - Creating the patch against what you've committed so far might be useful.
+    Use `git diff` from the repository root, or `git diff --relative` within
+    the benchmark directory. Copy the output into the `N-*.patch` file. - Do a test run with an `IncrPatched` scenario to make sure the patch
+    applies correctly, e.g. `target/release/collector bench_local +nightly
+--id Test --profiles=Check --scenarios=IncrPatched
+--include=$NEW_BENCHMARK`
```

Comment:
Why all the whitespace changes in this file? Some of them seem reasonable, but this one isn't. If they are deliberate, they would be best done in a separate commit.

Reply:
Oops, that was an unintentional autoformat done by my IDE. Reverted the whitespace changes.